integration-tests/gradle/projects/it-wasm-0/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+import org.jetbrains.dokka.gradle.DokkaTask
```

Comment:
About the directory name `it-wasm-0`. It does look in line with the majority of other IT projects, but the `0` is not descriptive, so I'm persuing to get away from this 0-1-2 naming pattern.

Wdyt about naming it `it-wasm-basic` instead?